### PR TITLE
Add RF to BO SOFB

### PR DESCRIPTION
**Pull request by fernandohds564**

### CODE REVIEW

File: siriuspy/siriuspy/sofb/csdev.py [hunk 1]
```diff
@@ -804,6 +847,32 @@ def get_sofb_database(self, prefix=''):
         dbase.update(self._add_prefix(db_ring, prefix))
         return dbase
 
+    def get_corrs_database(self, prefix=''):
+        """Return OpticsCorr-Chrom Soft IOC database."""
```
Comment (anacso17) [inline siriuspy/siriuspy/sofb/csdev.py:851]: wouldn't it be `"Return SOFB correctors database."` ?

File: siriuspy/siriuspy/sofb/csdev.py [hunk 2]
```diff
@@ -804,6 +847,32 @@ def get_sofb_database(self, prefix=''):
...
+        db_ring = {
+            'KickRF-Mon': {
+                'type': 'float', 'value': 1, 'unit': 'Hz', 'prec': 2},
+            }
+        dbase = super().get_corrs_database(prefix=prefix)
+        dbase.update(self._add_prefix(db_ring, prefix))
+        return dbase
+
+    def get_respmat_database(self, prefix=''):
+        """Return OpticsCorr-Chrom Soft IOC database."""
```
Comment (anacso17) [inline siriuspy/siriuspy/sofb/csdev.py:861]: wouldn't it be `"Return SOFB response matrix database."` ?

File: siriuspy/siriuspy/sofb/csdev.py [hunk 3]
```diff
@@ -791,6 +795,45 @@ def __init__(self, acc):
     def get_sofb_database(self, prefix=''):
         """Return OpticsCorr-Chrom Soft IOC database."""
```
Comment (anacso17) [inline siriuspy/siriuspy/sofb/csdev.py:796]: idem here

File: siriuspy/siriuspy/sofb/csdev.py (hunk 1 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/sofb/csdev.py]: right, done!

**Review by anacso17 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Review by xresende — APPROVED** [review]

**Pull request merged**